tests/Bonsai.ML.LinearDynamicalSystems.Tests/WorkflowHelper.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using Bonsai.Expressions;
 using System.Reactive.Linq;
+using System.Threading.Tasks;
```

Comment:
Is this a missing `using` statement? I wonder why it wasn't triggering build CI before, or maybe we just weren't building?

Reply:
It used to come from a global using statement.